engine/src/main/java/org/terasology/world/internal/WorldProviderCoreImpl.java

```diff
@@ -67,7 +67,7 @@
 
     private final List<WorldChangeListener> listeners = Lists.newArrayList();
 
-    private final Map<Vector3i, BlockChange> blockChanges = Maps.newHashMap();
+    private final Map<org.terasology.math.geom.Vector3i, BlockChange> blockChanges = Maps.newHashMap();
```

Comment:
we can't migrate this?

Reply:
hm... Let me double check how much this actually affects, but because of type erasure it is not possible to do this trick with the second method in the interface and forwarding with default implementation.
Migrating this method would also mean to migrate all callers at the same time...

Reply:
Yep, there's a long rat tail following that change. I'd like to do that in a follow-up PR.

It will also affect other places in the code, such as `world.block.structure.*`....

Reply:
that sounds fine. we can do it in another pass.